v3/src/components/graph/utilities/graph-utils.ts

```diff
@@ -675,3 +679,49 @@ export const curveBasis = (points: Point[]) => {
   path += pathBasis(p2, p3, p3, p3)
   return path
 }
+
+const axisPlaceToOrientationMap: Record<GraphPlace, AxisOrientation> = {
+  "bottom": "horizontal",
+  "top": "horizontal",
+  "left": "vertical",
+  "rightCat": "vertical",
+  "rightNumeric": "vertical",
+  "yPlus": "vertical",
+  "plot": "horizontal",
+  "legend": "horizontal"
```

Comment:
`"plot"` and `"legend"` don't really have an orientation, so these might be better left `undefined`.